jumpscale/clients/taiga/taiga.py

```diff
@@ -217,3 +224,302 @@ def _get_project_status(project_statuses, status):
+            )
+
+    def export_issues_per_user(self, user_name=None, path="/tmp/issues_per_user.md", with_description=False):
+        """Export all the issues per specif user in a markdown file, if no user given
+        the method export issues for the current logged in user
+        
+        Args:
+            path (str): The path of exported markdown file.
+            user_name (str): The name of the user we want to get his issues.
+            
+        """
+        if user_name:
+            selected_user_name = user_name
+        else:
+            selected_user_name = self.username
+        user_id = self._get_user_id(selected_user_name)
+        self.text = f"""
+### {selected_user_name} Issues \n
+"""
+        greenlet = gevent.spawn(self.map_render_issues_per_user, user_id, with_description)
+        gevent.joinall([greenlet])
+
+        j.sals.fs.write_file(path=path, data=self.text)
+
+    def __group_data(self, data, grouping_attribute="project"):
+        """Get similar objects together with same specific attribute.
+        
+        Args:
+            data(list): list of data to be grouped
+            grouping_attribute (str): the property used to group the data together
+        
+        Returns:
+            grouped_data(defaultdict) : dict of list containing the data which are similar grouped together 
+        """
+        grouped_data = defaultdict(list)
+        for obj in data:
+            grouped_data[obj.get(grouping_attribute)].append(obj)
+        return grouped_data
+
+    def __get_stories_required_data(self):
+        """Get the required data from user stories to be used later in export(render)
+
+        Returns:
+            List: List of stories required details to be used in export(render).
+        """
+        all_stories_template = list()
+        users_stories = self.list_all_user_stories()
+        for story in users_stories:
+            single_story_template = dict()
+            single_story_template["subject"] = story.subject
+            if story.assigned_to_extra_info:
+                single_story_template["assigned"] = story.assigned_to_extra_info.get("full_name_display", "unassigned")
+                single_story_template["is_active"] = story.assigned_to_extra_info.get("is_active", "unknown")
+            tasks = list()
+            for task in story.list_tasks():
+                tasks.append(task.subject)
+            single_story_template["tasks"] = tasks
+            all_stories_template.append(single_story_template)
+        return all_stories_template
+
+    def export_all_user_stories(self, path="/tmp/all_stories.md"):
+        """Export all the user stories in a markdown file
+        
+        Args:
+            path (str): The path of exported markdown file.
+            
+        """
+        text = f"""
+### All User Stories \n
+"""
+        all_stories_template = self.__get_stories_required_data()
+
+        for story in all_stories_template:
```

Comment:
I still think using a template **file** is better than building this markdown like this.